### ISSUE TITLE
Wrong python and additional build option 

### ISSUE THREAD

**Issue by JohanMabille** [xtensor-stack/xtensor-python#239]
The following argument can be passed to cmake in order to fix wrong python detection:

`-DPYTHON_EXECUTABLE:FILEPATH=path_to_python`.

I guess it is equivalent to -DPYTHON_EXECUTABLE=`which python` but I'm not sure.

This should be added to the build section of the documentation (or in a troubleshooting section).

**Comment by SylvainCorlay** [comment]
Yes, we do this in the build of the conda recipe. This should definitely be documented.
